Specs/Scene/TilesetMetadataSpec.js

```diff
@@ -292,12 +299,13 @@ describe("Scene/TilesetMetadata", function () {
       class: {
         properties: {
           height: {
+            type: "SCALAR",
             componentType: "FLOAT32",
-            optional: true,
+            required: false,
```

Comment:
Well, right now these are not used, only `default`. I think I'll hold off until a future PR, because the scope for required/noData/etc. is a little ill-defined at present and I don't want to bloat this PR too much.